docs/runbooks/CDINoDefaultStorageClass.md

```diff
@@ -0,0 +1,38 @@
+# CDINoDefaultStorageClass
+<!-- Edited by agilboa, 4 Dec 2023 -->
+
+## Meaning
+
+This alert fires when there is no default (Kubernetes or virtualization) storage class, and a data volume is pending for one.
+
+A default virtualization storage class has precedence over a default Kubernetes storage class for creating a VirtualMachine disk image.
+
+## Impact
+
+If there is no default (k8s or virt) storage class, a data volume that requests a default storage class (storage class not explicitly specified) will be pending for one.
+
+## Diagnosis
+
+Get the default Kubernetes storage class by running the following command:
+
+```bash
+$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubernetes.io/is-default-class"=="true")|.name'
+```
+
+Get the default virtualization storage class by:
+
+```bash
+$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+```
+
+## Mitigation
+
+Ensure that there is one storage class that has the default (k8s or virt) storage class annotation.
```

Comment:
@arnongilboa  Is there a documentation for setting a default storage class?

Reply:
Added